src/main/java/il/ac/bgu/cs/bp/bpjs/BPjs.java

```diff
@@ -144,5 +144,12 @@ public static void setLogDuringVerification(boolean logDuringVerification) {
         BPjs.logDuringVerification = logDuringVerification;
     }
     
+    public static void getPomVersion(){
+        Package mainPackage = Main.class.getPackage();
```

Comment:
Can we use the `BPjs` class here, instead of `Main`? This is because `BPjs` is the central entry point for meta-data, while `Main` is just a CLI entry point.